src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -583,9 +661,39 @@ export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({
               </AccordionPanel>
             </AccordionItem>
           )}
+          {userContext.apiType === "SQL" && userContext.authType === AuthType.AAD && (
```

Comment:
For Fabric, the client needs to go a specific endpoint that the Fabric workload can intercept.
```suggestion
          {userContext.apiType === "SQL" && userContext.authType === AuthType.AAD && !isFabric() && (
```

Reply:
Added configContext.platform !== Platform.Fabric condition.